src/index.ts

```diff
@@ -183,12 +183,16 @@ export default class OtaClient {
     async getLanguageTranslations(languageCode?: string): Promise<LanguageTranslations[]> {
         const language = this.getLanguageCode(languageCode);
         const files = await this.listFiles();
-        return Promise.all(
+        const languageTranslations = await Promise.all(
             files.map(async file => {
                 const content = await this.getFileTranslations(file, language);
-                return { content, file } as LanguageTranslations;
+                if (!content) {
```

Comment:
instead of completly removing file from the final array, let's just leave empty `null` content.  So change 
```typescript
export interface LanguageTranslations {
    file: string;
    content: string | any;
}
```
to
```typescript
export interface LanguageTranslations {
    file: string;
    content?: string | any;
}
```
and revert all changes here

Reply:
@ImRodry Thanks for support! I left one comment regarding that case of missing files.

Reply:
null does not mean optional, and since you type it as any, changing the type is redundant. I can improve the type by making it a Record

Reply:
Yep, but the point is to avoid silent ignore of the file and let clients to decide how to handle it

Reply:
Maybe this will work better:
```typescript
export interface LanguageTranslations {
    file: string;
    content?: string | object;
}
```
and use `undefined` instead of `null`

Reply:
That’s extremely bad type practice to use explicit undefined, type undefined as optional and use the object type. I’ll strengthen these types a bit in a way that makes sense

Reply:
Agreed. Just want to emphases that file should be still there but without content. Still think that can be achieved with optimal field but yep without explicit undefined. Just do not set this field and simple return e.g. `{ file }`if method did not return anything (null).

Reply:
Ended up setting content to an explicit null and strengthening the types to Record<string, any>. I also highly recommend updating your axios version as the one you have installed currently defaults all types to unknown, which was reverted in 0.24.0 I believe.

Reply:
Will do with a fresh release